packages/zoltan2/test/connectivity/meshData/README.txt

```diff
@@ -0,0 +1,42 @@
+Status:  8/8/18
+
+File formats:
+
+*.quad.msh -- mesh files with quad elements
```

Comment:
ant*km-borders.quad.msh is a member of *.quad.msh, but has different format (as shown below)

Reply:
Also, there are no other *.quad.mesh files besides *-borders.quad.msh

Reply:
This documentation was written before we made the switch to MatrixMarket files, so this part is outdated.